lib/config.js

```diff
@@ -13,8 +13,9 @@ function Config (process) {
   this.httpProxy = process.env.IED_HTTP_PROXY || process.env.HTTP_PROXY || null
   this.httpsProxy = process.env.IED_HTTPS_PROXY || process.env.HTTPS_PROXY || null
   this.requestRetries = parseInt(process.env.IED_REQUEST_RETRIES, 10) || 5
-  this.sh = process.env.IED_SH || (process.platform === 'win32' ? process.env.comspec || 'cmd' : 'sh')
-  this.shFlag = process.env.IED_SH_FLAG || (process.platform === 'win32' ? '/d /s /c' : '-c')
+  this.sh = process.env.IED_SH || (isWindows ? process.env.comspec || 'cmd' : 'sh')
+  this.shFlag = process.env.IED_SH_FLAG || (isWindows ? '/d /s /c' : '-c')
```

Comment:
Yeah I did.

It solves the administrator requirement when installing and gets things installing to `node_modules`.

I couldn't get a few things working still like some packages don't work after installing (whereas they do when installing with `npm`). I still have to investigate what's causing it a little bit more as the files are definitely on disk.